gobuffalo/azuredeploy.parameters.json

```diff
@@ -1,5 +1,12 @@
 {
     "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
-    "parameters": {}
+    "parameters": {

```

Comment:
will need to add GEN-PASSWORD for the pw

Reply:
Fixed locally.